src/convertToObject.js

```diff
@@ -5,8 +5,61 @@
  *
  * @return {object}
  */
-function convertToObject(sourceString) {
+function convertToObject(
+  sourceString = `
+background-color:      #fff;
+-webkit-border-radius: 5px;
+border-radius     : 5px;
+border: 1px solid #e8e8e8;
+-webkit-box-sizing: border-box;
+box-sizing: border-box;
+clear   : both       ;
+cursor: pointer;
+float: left;
+font-family: inherit;
+    font-size: 14px;
+font-weight: 400;
+height: 42px;
+line-height:    40px;
+outline: 0;
+padding-left    : 18px;
+padding-right: 30px;
+;
+
+;
+position: relative;
+
+
+text-align: left !important;
+-webkit-transition: all .2s ease-in-out;
+transition: all .2s ease-in-out;
+-moz-user-select: none;
+-ms-user-select: none;
+user-select: none;
+
+
+white-space: nowrap;
+width: auto;
+`,
+) {
```

Comment:
The default parameter for `sourceString` is a large string literal. This is not a good practice as it clutters the function signature. Consider removing the default value and ensuring that the function is always called with an appropriate argument.